src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -297,7 +305,12 @@ export class HuiCardPicker extends LitElement {
     }
 
     return html`
-      <div class="card" @click="${this._cardPicked}" .config="${cardConfig}">
+      <div class="card">
+        <div
+          class="overlay"
+          @click=${this._cardPicked}
+          .config=${cardConfig}
```

Comment:
You didn't have to move this, but it it fine.

Reply:
Gotcha I guess I didnt test with out changing. I just assumed it would be above it and not register the click on the lower element